source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContentHoldingUIView.mm

```diff
@@ -104,6 +129,63 @@ - (void)layoutSubviews
                 [layer removeFromSuperlayer];
             }
         }
+        // The content view is UIImageView
+    } else {
+        CGSize frameSize = self.frame.size;
+        if (frameSize.width < self.imageProperties.contentSize.width && _imageView.image) {
```

Comment:
rather than having `&& _imageView.image` here and on line 146, perhaps just change 133 to `} else if (_imageView.image) {`?

Reply:
thanks; will do.